use conform and exrm from upstream

0xAX opened the pull request.

0xAX committed:
use conform and exrm from upstream

umatomba commented on mix.exs:
```diff
@@ -28,10 +28,12 @@ defmodule UniMux.Mixfile do
-     {:metricman, github: "xerions/metricman", branch: "master"},
-     {:exlager, github: "xerions/exlager"},
+     {:metricman, github: "0xAX/metricman", branch: "upstream-conform", override: true},
+     {:exlager, github: "0xAX/exlager", branch: "upstream-conform", override: true},
```
Can we use upstreams here?

0xAX replied:
Yes, forgot to update unimux, will do it

0xAX commented:
@umatomba I've updated also I've removed elixir 1.0.5 from CI because new exrm uses Mix.Project.compile/2 which is occurred only in the elixir 1.1.0
